src/verify.ts

```diff
@@ -53,9 +49,7 @@ export class Verifier {
     const publicKey = await this.getPublicKey(bundle);
 
     verifyArtifactSignature(bundle, publicKey, data);
-    verifyTLogSET(bundle, this.tlogKeys);
-    verifyTLogBodies(bundle);
-    verifyTLogIntegratedTime(bundle);
+    verifyTLogEntries(bundle, this.tlogKeys);
```

Comment:
This is the primary purpose of this PR -- encapsulate these three verification steps behind a single function.